spyder/config/appearance.py

```diff
@@ -0,0 +1,641 @@
+import os
```

Comment:
This is missing the header we add to all files in our source code. Please look at any other file and copy the first ten lines or so, which contain our copyright and the `coding` statement.

Reply:
And please also add a docstring to this module, below those comments and above this line.